core/src/test/java/org/elasticsearch/common/PriorityTests.java

```diff
@@ -82,11 +81,11 @@ public void testCompareTo() {
         for (Priority p : Priority.values()) {
             assertEquals(0, p.compareTo(p));
         }
-        List<Priority> shuffeledAndSorted = new ArrayList<>(Priority.values());
+        List<Priority> shuffeledAndSorted = Arrays.asList(Priority.values());
         Collections.shuffle(shuffeledAndSorted, random());
         Collections.sort(shuffeledAndSorted);
         for (List<Priority> priorities : Arrays.asList(shuffeledAndSorted,
-            Priority.values())) { // #values() guarantees order!
+            Arrays.asList(Priority.values()))) { // #values() guarantees order!
```

Comment:
You don't have to be sorry. I'm just saying we do it all the time. If we shouldn't do it we shouldn't do it.